collect_app/src/main/java/org/odk/collect/android/openrosa/api/FormListApi.java

```diff
@@ -1,10 +1,15 @@
 package org.odk.collect.android.openrosa.api;
 
+import java.io.InputStream;
 import java.util.List;
 
 public interface FormListApi {
 
     List<FormListItem> fetchFormList() throws FormApiException;
 
     ManifestFile fetchManifest(String manifestURL) throws FormApiException;
+
+    InputStream fetchForm(String formURL) throws FormApiException;
```

Comment:
`fetchForm` and `fetchMediaFile` happen to have the same implementation but it felt keeping the separate was the right thing to do™️ as the call sites don't really know or care that they involve the same work.